darwin/importer/importer.py

```diff
@@ -278,18 +284,42 @@ def _resolve_annotation_classes(
     return local_classes_not_in_dataset, local_classes_not_in_team
 
 
-def _get_team_properties_annotation_lookup(client, team_slug):
+def _get_team_properties_annotation_lookup(
+    client: "Client", team_slug: str
+) -> Tuple[Dict[Tuple[str, Optional[int]], FullProperty], Dict[str, FullProperty]]:
```

Comment:
Some of these types are looking a bit complicated, could be nice when doing the refactor to define some more types

Reply:
Totally agree, will take place as part of DAR-4295